tekton/src/tasks/gitops/gitops-deprovision-app-config.yml.j2

```diff
@@ -60,10 +63,18 @@ spec:
         value: $(params.force_removal)
       - name: MAS_APP_ID
         value: $(params.mas_app_id)
+      - name: OCP_SERVER
```

Comment:
same as above...
why we're using `$(params.cluster_url)` to set `OCP_SERVER`?
should this be setting `CLUSTER_URL` or the other way around?

Reply:
same as above. The value set in param cluster_url is set to env variable OCP_SERVER